### PR TITLE
chore: apply stale probot and change uglifyjs-webpack-plugin to terser-webpack-plugin

### PR DESCRIPTION
**Pull request by jungeun-cho**
### Please check if the PR fulfills these requirements
- [x] It's submitted to right branch according to our branching model
- [x] It's right issue type on title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has description for the breaking change

### Description
* chore: apply stale probot
* docs: add issue templates
* chore: change `uglifyjs-webpack-plugin`(deprecated) to `terser-webpack-plugin
    * Fixed security alerts for a vulnerable dependency (for `serialize-javascript`)

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

**Comment by jungeun-cho** [comment]
ok to test

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
```
Comment (dongsik-yoo) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Is this official template or new one?

Comment (jungeun-cho) [reply .github/ISSUE_TEMPLATE/bug_report.md]: It is not an official template.
However, it is already applied to Chart, Editor and Grid of TOAST UI Product.
What do you think?

Comment (dongsik-yoo) [reply .github/ISSUE_TEMPLATE/bug_report.md]: What is the issue type? I think this is enough to fullfill a form.

**Pull request merged**